content/sensu-go/5.19/reference/backend.md

```diff
@@ -1243,12 +1244,13 @@ event-log-file: "/var/log/sensu/events.log"{{< /highlight >}}
 ### Log rotation
 
 Event logging supports log rotation via the _SIGHUP_ signal.
-First, rename (move) the current log file.
+To rotate event logs, first rename (move) the current log file.
 Then, send the _SIGHUP_ signal to the sensu-backend process so it creates a new log file and starts logging to it.
 
-Here are some log rotate sample configurations:
+#### Log rotation for systemd logs
```

Comment:
```suggestion
Because event log files can grow very quickly for larger Sensu installations, we recommend using `logrotate` to automatically rotate log files.

#### Log rotation for systemd
```